index.html

```diff
@@ -1,1046 +1,1078 @@
+      versatility and wide adoption of JWS make it an essential
+      component for verifying the authenticity and integrity of
+      JSON-based messages in various applications, including
+      authentication, authorization, and secure data exchange. These
+      properties make JWS especially well suited to securing the
+      [[VC-DATA-MODEL]] and through this to provide a means of
+      supporting the three party model (issuer, verifier, holder) for
+      exchange of credentials enabled in verifiable credenials.
+    </p>
+    <p>
+      JSON Web Token (JWT) [[RFC7519]] is a widely-used means of
+      expressing
+      <a data-cite="VC-DATA-MODEL#claims">claims</a> to be transferred
+      between two parties. Providing a representation of the
+      Verifiable Credentials Data Model for JWT allows existing
+      systems and libraries to participate in the ecosystem described
+      in Section <a
+      data-cite="VC-DATA-MODEL#ecosystem-overview">ecosystem
+      overview</a>. A JWT encodes a set of
+      <a data-cite="VC-DATA-MODEL#claims">claims</a> as a JSON object
+      that is contained in a JSON Web Signature (JWS) [[RFC7515]]
+      and/or JSON Web Encryption (JWE) [[?RFC7516]]. For this
+      specification, the use of JWE is out of scope.
+    </p>
+    <p>
+      CBOR Object Signing and Encryption (COSE) [[RFC8152]] is a
+      specification that defines a framework for signing and
+      encrypting CBOR (Concise Binary Object Representation) [RFC8949]
+      data structures. COSE provides a standardized way to secure the
+      integrity, authenticity, and confidentiality of CBOR-encoded
+      information. It offers a flexible and extensible set of
+      cryptographic options, allowing for a wide range of algorithms
+      to be used for signing and encryption.
 
+      COSE supports two main operations: signing and encryption. For
+      signing, COSE allows the creation of digital signatures over
+      CBOR data using various algorithms such as HMAC, ECDSA, or
+      EdDSA. These signatures provide assurance of data integrity and
+      authenticity. COSE also supports encryption, enabling the
+      confidentiality of CBOR data by encrypting it with symmetric or
+      asymmetric encryption algorithms. Our use case here, with
+      verifiable credentials is focused on signing, primarily through
+      use of COSE Sign1 operations.
+    </p>
+  </section>
 
+  <section>
+    <h2>Securing the Core VC Data Model</h2>
+    <p>
+      The use of Verifiable Credentials often involves the
+      representation and exchange of structured data in the form of
+      JSON-LD as this is the format of the core data model. While
+      JSON-LD provides a flexible and extensible format for describing
+      data, it is important to note that it also provides a linkage
+      between the data structure and semantic meaning of data.
+    </p>
+    <p>
+      This section outlines how JSON-LD encoded claimset can be secured
```

Comment:
```suggestion
      This section outlines how JSON-LD encoded payload can be secured
```

Reply:
one could argue that JSON-LD is not a claimset, since its not JSON... 
... or they could argue that it is a claimset, since it is JSON... 
see https://www.w3.org/TR/json-ld11/#relationship-to-rdf

Based on previous formal objections:

https://www.w3.org/2022/03/did-fo-report.html

I suggest we add an advisement here, and point to https://github.com/w3ctag/design-principles/issues/239